### ISSUE TITLE
Error processing request

### ISSUE THREAD

**Issue by Jeoffreybauvin** [lmenezes/cerebro#457]
Hi,

I have an issue with Cerebro 0.9.1 and Elasticsearch 5.6.6 :

```
[2020-06-04T14:47:40.030+02:00][error][application              ] Error processing request [path: /cerebro/nodes, body: {"password":"xxxxxx","host":"elasticsearch","username":"jbauvin"}]
play.api.libs.json.JsResultException: JsResultException(errors:List((,List(JsonValidationError(List('attributes' is undefined on object: {"name":"node-002","transport_address":"10.2.231.40:9300","host":"10.2.231.40","ip":"10.2.231.40","version":"5.6.8","build_hash":"688ecce","roles":["master","data"],"os":{"refresh_interval":"1s","refresh_interval_in_millis":1000,"name":"Linux","arch":"amd64","version":"3.16.0-7-amd64","available_processors":3,"allocated_processors":3},"jvm":{"pid":126057,"version":"1.8.0_60","vm_name":"Java HotSpot(TM) 64-Bit Server VM","vm_version":"25.60-b23","vm_vendor":"Oracle Corporation","start_time":"2019-03-29T14:51:37.796Z","start_time_in_millis":1553871097796,"mem":{"heap_init":"1.7gb","heap_init_in_bytes":1853882368,"heap_max":"1.7gb","heap_max_in_bytes":1827733504,"non_heap_init":"2.4mb","non_heap_init_in_bytes":2555904,"non_heap_max":"0b","non_heap_max_in_bytes":0,"direct_max":"1.7gb","direct_max_in_bytes":1827733504},"gc_collectors":["ParNew","ConcurrentMarkSweep"],"memory_pools":["Code Cache","Metaspace","Compressed Class Space","Par Eden Space","Par Survivor Space","CMS Old Gen"],"using_compressed_ordinary_object_pointers":"true","input_arguments":["-Dfile.encoding=UTF-8","-Dio.netty.noKeySetOptimization=true","-Dio.netty.noUnsafe=true","-Dio.netty.recycler.maxCapacityPerThread=0","-Djava.awt.headless=true","-Djna.nosys=true","-Dlog4j.shutdownHookEnabled=false","-Dlog4j2.disable.jmx=true","-XX:+AlwaysPreTouch","-XX:+HeapDumpOnOutOfMemoryError","-XX:+PrintGCApplicationStoppedTime","-XX:+PrintGCDateStamps","-XX:+PrintGCDetails","-XX:+PrintTenuringDistribution","-XX:+UseCMSInitiatingOccupancyOnly","-XX:+UseConcMarkSweepGC","-XX:+UseGCLogFileRotation","-XX:-OmitStackTraceInFastThrow","-XX:CMSInitiatingOccupancyFraction=75","-XX:GCLogFileSize=64m","-XX:NumberOfGCLogFiles=32","-Xloggc:/var/log/elasticsearch/node-002/gc.log","-Xms1768m","-Xmx1768m","-Xss1m","-Des.path.home=/usr/share/elasticsearch"]}}),WrappedArray())))))
	at play.api.libs.json.JsReadable.$anonfun$as$2(JsReadable.scala:25)
	at play.api.libs.json.JsError.fold(JsResult.scala:66)
	at play.api.libs.json.JsReadable.as(JsReadable.scala:24)
	at play.api.libs.json.JsReadable.as$(JsReadable.scala:23)
	at play.api.libs.json.JsUndefined.as(JsLookup.scala:193)
```

**Comment by oaxiento** [comment]
+1 from me as well. We're having the same issue with 0.9.1. Version 0.9.0 works just fine!

**Comment by lmenezes** [comment]
this is a duplicate of #448 which should be solved now (0.9.2). Sorry for the trouble :)

**Comment by oaxiento** [comment]
@lmenezes Thanks!
